### ISSUE TITLE
Learner files starting with underscore not included

### ISSUE THREAD

**Issue by jimafisk** [plentico/plenti#195]
The default "learner" start includes files that start with an underscore, for example `content/blog/_blueprint.json`. However, when you create a new site with `plenti new site my-site` this file is missing.

It appears that Go's embed does not included underscores by default: https://github.com/golang/go/issues/43854

**Comment by jimafisk** [comment]
The fix is to just add a comment like this for your embedded FS:

```go
//go:embed all:pattern
```

For example:
```go
//go:embed all:defaults/starters/learner/*
var defaultsLearnerFS embed.FS
```

**Comment by jimafisk** [comment]
Should be fixed in [v0.5.8](https://github.com/plentico/plenti/releases/tag/v0.5.8)
